.evergreen/config.yml

```diff
@@ -1378,7 +1379,17 @@ functions:
         binary: bash
         args:
           - .evergreen/fetch-drivers-tools.sh
-
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        include_expansions_in_env:
+          - DRIVERS_TOOLS
+        binary: bash
+        args:
+          - .evergreen/find-python3.sh
```

Comment:
This addition means I don't have to wrap the server startup/shutdown in shell scripts (and might be useful for similar later things).